core/src/main/java/org/apache/carbondata/core/datamap/DataMapChooser.java

```diff
@@ -93,6 +93,48 @@ public DataMapExprWrapper choose(CarbonTable carbonTable, FilterResolverIntf res
         resolverIntf);
   }
 
+  /**
+   * Return a chosen datamap based on input filter. See {@link DataMapChooser}
+   */
+  public DataMapExprWrapper chooseFG(CarbonTable carbonTable, FilterResolverIntf resolverIntf)
```

Comment:
I think better to rename to `chooseFGDataMap`

Reply:
ok